### PR TITLE
Remove `@data` Attribute

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
In this PR I:
1) remove `data` attribute usage.
2) add `-` dash to the `00-` bytes
3) upgrade the `eo` version  up to `0.43.2`.

All these changes are made in the same PR because they are related and we can't remove `data` attribute without upgrading `eo`. And we can't upgrade `eo` without adding `-`. 

Related to #850, #908

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you review these changes, please?

File: src/main/java/org/eolang/jeo/representation/directives/DirectivesProgram.java [hunk 1]
```diff
@@ -117,14 +117,73 @@ public Iterator<Directive> iterator() {
...
-            .add("errors").up()
-            .add("sheets").up()
-            .add("license").up()
+            .add("errors")
+            .add("error").attr("severity", "warning").set(DirectivesProgram.help()).up()
+            .up()
+            .add("sheets")
+            .add("sheet").set(DirectivesProgram.help()).up()
+            .up()
+            .add("comments")
```
Comment (yegor256) [inline src/main/java/org/eolang/jeo/representation/directives/DirectivesProgram.java:126]: @volodya-lombrozo just don't add `comments` at all and you'll be fine. No need to create this fake data.

File: src/main/java/org/eolang/jeo/representation/directives/DirectivesProgram.java [hunk 2]
```diff
@@ -117,14 +117,73 @@ public Iterator<Directive> iterator() {
             .add("listing")
             .set(this.listing)
             .up()
-            .add("errors").up()
-            .add("sheets").up()
-            .add("license").up()
+            .add("errors")
```
Comment (yegor256) [inline src/main/java/org/eolang/jeo/representation/directives/DirectivesProgram.java:120]: @volodya-lombrozo same here, don't add `sheets` and `errors`

**Review by yegor256 — CHANGES_REQUESTED** [review]
@volodya-lombrozo looks cool! A few comments above

**Comment by volodya-lombrozo** [comment]
@yegor256 Indeed. It helped. Thank you, can you take one more look, please?

**Review by yegor256 — APPROVED** [review]
@volodya-lombrozo LGTM

**Comment by volodya-lombrozo** [comment]
@rultor merge

**Pull request merged**